src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String defaultColor = "white";
```

Comment:
```suggestion
    private static final String DEFAULT_COLOR = Color.WHITE.name().toLowerCase();
```

Read about constants naming